spec/specification/linter_spec.rb

```diff
@@ -176,9 +176,14 @@ def result_should_include(*values)
         result_should_include('license', 'unrecognized `name` key')
       end
 
-      xit 'checks the source for unknown keys' do
```

Comment:
this test has been disabled for a long time, decided to update it while I was here